CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+## Questions
+If you have questions, please open the issue and await our response.
+If you are seeking for help as user, please try Stack Overflow.
+
+## Issue Labeling
+
+| Label | Description |
+| ------|-------------|
+| docs | Indicate documentation development necessary |
+| feat | Indicate feature development |
+| CI | Indicate Continuous integration development |
+| refactor | Indicate code refactor |
+| remove | Indicate feature removal |
+| fix | Indicates an unexpected problem or unintended behavior |
+| duplicate | Indicates similar issues or pull requests |
+| enhancement | Indicates new feature requests |
+| good first issue | Indicates a good issue for first-time contributors |
+| help wanted | Indicates that a maintainer wants help on an issue or pull request |
+| invalid | Indicates that an issue or pull request is no longer relevant |
+| question | Indicates that an issue or pull request needs more information |
+| wontfix | Indicates that work won't continue on an issue or pull request |
+
+## Report a bug fix
+To report a fix, follow the issue template, if possible attach a [codesandbox](https://codesandbox.io/) or [jsBin](https://codesandbox.io/) with the error.
+
+## Proposal a Feature
+Open a problem, mark it as other, and describe the current scenario, its problems, and how your resource solves this problem.
+
+## Status Check List
+- danger system
+  - prettier code format
+  - jsdocs coverage
```

Comment:
i think this should be 2 topics, one for jsdocs and one for code coverage, with links for techs maybe?